source/includes/agents/_20-show.md.erb

```diff
@@ -3,14 +3,14 @@
 ```shell
 $ curl 'https://ci.example.com/go/api/agents/adb9540a-b954-4571-9d9b-2f330739d4da' \
       -u 'username:password' \
-      -H 'Accept: application/vnd.go.cd.v3+json'
+      -H 'Accept: application/vnd.go.cd.v4+json'
 ```
 
 > The above command returns JSON structured like this:
 
 ```http
 HTTP/1.1 200 OK
-Content-Type: application/vnd.go.cd.v3+json; charset=utf-8
+Content-Type: application/vnd.go.cd.v4+json; charset=utf-8
```

Comment:
Could you update the example below with how a building agent with `build_details` looks like?